docs/tutorials/install.md

```diff
@@ -80,46 +80,46 @@ git pull origin main
 make install
 ```
 
-## Compile with CLevelDB support
+## Compile with RocksDB support
```

Comment:
were these commands tested on both Ubuntu and MacOS ? Or just a rename ? I am assuming these new commands and instructions would work fine.